server/venueless/core/models/world.py

```diff
@@ -143,6 +143,7 @@ class World(VersionedModel):
     )
     timezone = models.CharField(max_length=120, default="Europe/Berlin")
     feature_flags = JSONField(blank=True, default=default_feature_flags)
+    external_auth_url = models.CharField(max_length=255, default="")
```

Comment:
I think ``models.URLField`` would be a better choice. Also, the default should not be an empty string, but a real null-value, e.g.

```
models.URLField(null=True, blank=True)
```